File: VB/dxSample/Custom Layout Control/CustomLayoutConrol.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DevExpress.XtraLayout

Namespace dxSample
    Friend Class CustomLayoutControl
        Inherits LayoutControl

        Protected Overrides Sub CreateILayoutControlImplementor()
            MyBase.CreateILayoutControlImplementor()
        End Sub
        Protected Overrides Function CreateILayoutControlImplementorCore() As LayoutControlImplementor
            Return New CustomLayoutControlImplementor(Me)
        End Function
    End Class
End Namespace

File: CS/dxSample/Custom Layout Control/CustomLayoutConrol.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.XtraLayout;

namespace dxSample {
    internal class CustomLayoutControl : LayoutControl {
        protected override void CreateILayoutControlImplementor() {
            base.CreateILayoutControlImplementor();
        }
        protected override LayoutControlImplementor CreateILayoutControlImplementorCore() {
            return new CustomLayoutControlImplementor(this);
        }
    }
}
